schemas/products/hardwareModel.schema.tpl.json

```
@@ -0,0 +1,59 @@
+{
+    "_type": "https://openminds.ebrains.eu/ephys/HardwareModel",
+    "required": [
+        "deviceType",
+        "manufacturer",
+        "modelName"
+    ],
+    "properties": {
+        "internalIdentifier": {
+            "type": "string",
+            "_instruction": "Enter the identifier of this Device that is used within the corresponding data."
+        },
+        "deviceType": {
+            "_instruction": "Reference to a device type",
+            "_linkedTypes": [
+                "https://openminds.ebrains.eu/controlledTerms/DeviceType"
+            ]
+        },
+        "manufacturer": {
+            "type": "string",
+            "_instruction": "The manufacturer or supplier providing the item. If it is self-made please state 'custom-made'."
+        },
+        "modelName": {
+            "type": "string",
+            "_instruction": "The full name of this particular model as stated from the manufacturer."
+        },
+        "modelIdentifier": {
+            "_instruction": "Please indicate any persistent identifier asosiated with this hardware",
+            "_linkedTypes": [
+                "https://openminds.ebrains.eu/core/DOI",
+                "https://openminds.ebrains.eu/core/RRID"
+            ]
+        },
+        "serialNumber": {
+            "type": "string",
+            "_instruction": "specific for the model, provided by the manufacturer or institution."
+        },
```

```suggestion
        "AlternateIdentifier": {
            "type": "string",
            "_instruction": "inventory number or serial number specific for the model, provided by the manufacturer or institution."
        },
```
should this be an array to make serial number + inventory number possible ?

---

or maybe even "identificationNumber"'? if we make it an array, SerialNumer and InventoryNumber should probably become their own instance (as the other digital identifer). It is a valid argument to allow both @Peyman-N / @apdavison your view?

---

looking through the other schema again it should be : "manufacturerIdentifier" for serial number. if inventory number is chosen by the owner it is the "internalIdentifier" for openMINDS

---

I agree that the `internalIdentifier` and `InventoryNumber` are strongly related, but I see some cases that these two are different. Using the `InventoryNumber` in the data is not that common, as it is more for administrative purposes. But its advantage is that it is mostly unique. Anyway I argue `internalIdentifier` should be there while `InventoryNumber` is a nice to have.

---

then add "inventoryNumber" as an optional property

---

description suggestion: "Add the institutional inventory number (if it differs from the internal identifier)."